### ISSUE TITLE
bug: SES VerifyDomainDkim, SetIdentityMailFromDomain APIs do not seem to behave as expected

### ISSUE THREAD

**Issue by fs-td-eitaro-ueda** [localstack/localstack#4276]
### Is there an existing issue for this?

- [X] I have searched the existing issues

### Current Behavior

There are two seemingly wrong behavior regarding Amazon SES APIs:

**1. VerifyDomainDkim API**

Even when trying to verify DKIM for the existing identity, the API seems to add the same identity.

**2. SetIdentityMailFromDomain API**

It always returns 500. (Not supported yet?)

### Expected Behavior

**1. VerifyDomainDkim API**

When trying to verify DKIM for the existing identity, the API should try to verify DKIM without adding duplicated identity.

**2. SetIdentityMailFromDomain API**

The API should return result with right parameters (domain, mail from domain, default behavior on MX failure).

### How are you starting LocalStack?

With a docker-compose file

#### How are you starting localstack (e.g., `bin/localstack` command, arguments, or `docker-compose.yml`)

```yml
version: '3'
services:
  localstack:
    container_name: "${LOCALSTACK_DOCKER_NAME-localstack_main}"
    image: localstack/localstack
    ports:
      - "4566:4566"
      - "8080:8080"
    environment:
      - TMPDIR=/private$TMPDIR
      - DATA_DIR=/tmp/localstack/data
      - DEFAULT_REGION=us-west-2
      - LAMBDA_EXECUTOR=docker-reuse
      - LAMBDA_DOCKER_NETWORK=host
      - DOCKER_HOST=unix:///var/run/docker.sock
      - SERVICES=s3,sns,sqs,ses,lambda,cloudformation,sts,iam
    volumes:
      - /var/run/docker.sock:/var/run/docker.sock
      - ./docker/middleware/localstack/aws_config:/root/.aws/
```

#### Client commands (e.g., AWS SDK code snippet, or sequence of "awslocal" commands)

```bash
$ aws --endpoint-url=http://localhost:4566 ses list-identities
{
    "Identities": []
}
```

```bash
$ aws --endpoint-url=http://localhost:4566 ses verify-domain-identity --domain 'example.com'
{
    "VerificationToken": "QTKknzFg2J4ygwa+XvHAxUl1hyHoY0gVfZdfjIedHZ0="
}
```

```bash
$ aws --endpoint-url=http://localhost:4566 ses list-identities
{
    "Identities": [
        "example.com"
    ]
}
```

```bash
$ aws --endpoint-url=http://localhost:4566 ses verify-domain-dkim --domain 'example.com'
{
    "DkimTokens": [
        "vvjuipp74whm76gqoni7qmwwn4w4qusjiainivf6sf",
        "3frqe7jn4obpuxjpwpolz6ipb3k5nvt2nhjpik2oy",
        "wrqplteh7oodxnad7hsl4mixg2uavzneazxv5sxi2"
    ]
}
```

VerifyDomainDkim API seems to add the same identity (I think it should not add the same identity when already exists.)

```bash
$ aws --endpoint-url=http://localhost:4566 ses list-identities
{
    "Identities": [
        "example.com",
        "example.com"
    ]
}
```

SetIdentityMailFromDomain API returns 500. (I also have tried after deleting duplicated identity, but the result turned out to be the same.)

```bash
$ aws --endpoint-url=http://localhost:4566 ses set-identity-mail-from-domain --identity 'example.com' --mail-from-domain 'mail.example.com' --behavior-on-mx-failure 'UseDefaultValue'

Unable to parse response (syntax error: line 1, column 54), invalid XML received:
<!DOCTYPE HTML PUBLIC "-//W3C//DTD HTML 3.2 Final//EN">
<title xmlns="http://ses.amazonaws.com/doc/2010-01-31/">500 Internal Server Error</title>
<h1>Internal Server Error</h1>
<p>The server encountered an internal error and was unable to complete your request. Either the server is overloaded or there is an error in the application.</p>
```

### Environment

```markdown
- OS: MacOS
- LocalStack: latest
```

### Anything else?

AWS CLI Docs for reference:

https://docs.aws.amazon.com/cli/latest/reference/ses/verify-domain-dkim.html
https://docs.aws.amazon.com/cli/latest/reference/ses/set-identity-mail-from-domain.html

**Comment by usmangani1** [comment]
@fs-td-eitaro-ueda  
Thanks for reporting the issue. While the part 1(duplicate domain addition) is already fixed upstream. We are working on adding the SetIdentityMailFromDomain functionality.

**Comment by viren-nadkarni** [comment]
Fixed in[ 0.14.1](https://github.com/localstack/localstack/releases/tag/v0.14.1)
